spyder/plugins/variableexplorer/widgets/dataframeeditor.py

```diff
@@ -1803,6 +1804,16 @@ def __init__(
             triggered=self.show_preferences_dialog,
             register_action=False
         )
+        self.close_action = self.create_action(
+            name=DataframeEditorActions.Close,
+            icon=self.create_icon('close_pane'),
+            text=_('Close'),
+            triggered=self.reject,
+            shortcut=self.get_shortcut(DataframeEditorActions.Close),
+            context=Qt.WindowShortcut,
```

Comment:
```suggestion
            register_action=False,
```

- The previous code made the shortcut fail after clicking on the options menu.
- The current code is necessary to avoid a warning.